services/backend/src/modules/user/index.ts

```diff
@@ -21,6 +21,19 @@ export const getUser = async (id: string): Promise<RegularUser | null> => {
   return { ...user, id: snapshot.id }
 }
 
+export const getMenter = async (id: string): Promise<RegularUser | null> => {
+  const snapshot = await userCollection.doc(id).get()
+  const user = snapshot.data()
+  if (!user) {
+    return null
+  }
+  if (!user.menterId) {
+    throw new Error('Menter not found')
```

Comment:
`menterId` がない時は素直に null を返すで良いと思います！

```suggestion
    return null
```